src/perception/rai_whatisee/src/rai_whatisee_node.cpp

```diff
@@ -12,7 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#if defined(ROS_DISTRO_HUMBLE)
 #include <cv_bridge/cv_bridge.h>
+#elif defined(ROS_DISTRO_JAZZY)
```

Comment:
I checked. 
- Humble has only h
- Iron has both h and hpp
- Jazzy (and presumably later) have only hpp. 

So this should work with Iron (we don't support, but this doesn't increase the distance for users to make it work on Iron, which is good).